src/index.ts

```diff
@@ -78,6 +78,61 @@ if (stylesheet) {
   fs.copyFileSync('src/styles/index.css', `${output}/index.css`);
 }
 
+/**
+ * Process markdown for H1 (e.g: # headhing 1)
+ * @param data Data to be processed
+ * @return return array where h1 markdown is processed
+ */
+const processH1Markdown = (data: string[]): string[] => {
+  return data.map((text) => {
+    if (text.indexOf('# ') !== -1) {
+      if (text.substr(0, 2) === '# ') return `<h1>${text.substr(2)}</h1>`;
+    }
+    return `${text}`;
```

Comment:
Return `text` directly as text is already a string.

Reply:
removed